intro-HPC/leuven/available-modules.tex

```diff
@@ -0,0 +1,102 @@
+\begin{prompt}
+%\shellcmd{module av}%
+------------------- /apps/leuven/thinking/2014a/modules/all --------------------
+Abaqus/6.12-2
+Abaqus/6.13-3
+Abaqus/6.14-1
+accounting
+allinea-ddt/4.2
+annovar/2013Jun21
+ANSYS/13.0
+ANSYS/14.5
+ANSYS/15.0
+ant/1.9.3-Java-1.7.0_51
+Armadillo/4.320.0-foss-2014a-Python-2.7.6
+Armadillo/6.400.3-foss-2014a-Python-2.7.6
+Autoconf/2.69-foss-2014a
+Autoconf/2.69-GCC-4.7.2
+Autoconf/2.69-GCC-4.8.2
+Autoconf/2.69-iccifort-11.1.080
+Autoconf/2.69-intel-2014a
+Automake/1.14-GCC-4.8.2
+Automake/1.14-iccifort-11.1.080
+Automake/1.14-intel-2014a
+Bash/4.3
+Bash/4.3-GCC-4.8.2
+beagle-lib/20140304-intel-2014a
+BEAST/1.8.0
+BEAST/1.8.2
+BEAST/2.1.2
+bedtools/2.19.1-intel-2014a
+Biopython/1.66-foss-2014a-Python-2.7.6
+Bison/2.5-foss-2014a
+Bison/2.7.1-GCC-4.8.2
+Boost/1.54.0-foss-2014a
+Boost/1.55.0-foss-2014a
+Boost/1.55.0-foss-2014a-Python-2.7.6
+Boost/1.55.0-intel-2014a
+Boost/1.55.0-intel-2014a-Python-2.7.6
+Bowtie/1.0.1-intel-2014a
+Bowtie2/2.1.0-intel-2014a
+Bowtie2/2.2.1-intel-2014a
+BWA/0.7.5a-foss-2014a
+BWA/0.7.5a-intel-2014a
+bzip2/1.0.6-foss-2014a
+bzip2/1.0.6-intel-2014a
+%\dots% 
+\end{prompt}
```

Comment:
I understand that the output is shown like this, but maybe a dual-column format is better for use in the printed/pdf document, since it's illustrative and need not take up most of a page?